typed-rng/src/lib.rs

```diff
@@ -0,0 +1,234 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+//! Typed RNGs with support for trees seeds.
```

Comment:
```suggestion
//! Typed RNGs with support for tree seeds.
```

Reply:
Fixed, and expanded on this comment. Thanks!